cmd/vcluster/cmd/start.go

```diff
@@ -40,52 +47,53 @@ func NewStartCommand() *cobra.Command {
 				}
 			}()
 
+			// parse vCluster config
+			vClusterConfig, err := config.ParseConfig(startOptions.Config, os.Getenv("VCLUSTER_NAME"), startOptions.SetValues)
+			if err != nil {
+				return err
+			}
+
 			// execute command
-			return ExecuteStart(cobraCmd.Context(), vClusterOptions)
+			return ExecuteStart(cobraCmd.Context(), vClusterConfig)
 		},
 	}
 
-	options.AddFlags(cmd.Flags(), vClusterOptions)
-	pro.AddProFlags(cmd.Flags(), vClusterOptions)
+	cmd.Flags().StringVar(&startOptions.Config, "config", "/var/vcluster/config.yaml", "The path where to find the vCluster config to load")
+	cmd.Flags().StringArrayVar(&startOptions.SetValues, "set", []string{}, "Set values for the config. E.g. --set 'exportKubeConfig.secret.name=my-name'")
```

Comment:
not sure I understand what this is for. Isn't the config file final once it's in the vcluster pod?

Reply:
Yes exactly

Reply:
but why?